kbase-extension/static/kbase/custom/custom.js

```diff
@@ -107,7 +107,7 @@ define([
     'base/js/keyboard',
     'notebook/js/keyboardmanager',
     'notebook/js/cell',
-    'common/utils',
+    'common/cellUtils',
```

Comment:
This pattern will pop up over and over - functions that were only in `common/utils` and not `common/cellUtils` were copied over there, so just updating the import (and not what it was loaded as) is safe enough: each function is meant to be executed as a static utility call.